stix_shifter/stix_translation/src/modules/aws_cloud_watch_logs/data_mapping.py

```diff
@@ -0,0 +1,63 @@
+from os import path
+import glob
+import json
+from stix_shifter.stix_translation.src.modules.base.base_data_mapper import BaseDataMapper
+
+
+class DataMapper(BaseDataMapper):
+
+    def __init__(self, options):
+        mapping_json = options['mapping'] if 'mapping' in options else {}
+        basepath = path.dirname(__file__)
+        self.from_stix_files_cnt = self.json_files_to_fetch(path.abspath(
+            path.join(basepath, "json", "from_*.json")))
+        self.map_data = mapping_json or self.fetch_mapping()
+
+
+    @staticmethod
+    def json_files_to_fetch(file_path):
+        return glob.glob(file_path)
+
+    def fetch_mapping(self):
+        """
+        Fetches STIX-to-datasource mapping JSON from the module's from_stix_map.json file
+        :param basepath: path of data source translation module
+        :type basepath: str
+        """
+        map_data = {}
+        try:
+            for each_file in self.from_stix_files_cnt:
+                map_file = open(each_file).read()
+                map_data[path.basename(each_file)] = json.loads(map_file)
+            return map_data
```

Comment:
It looks like you are combining the mappings from all of your from_stix mapping files. I see right now you have mappings for guardduty and vpcflow. What happens when there are conflicting mappings, for example with the network-traffic object?

Reply:
The mappings are collected from each file and put it in a dictionary with file name as key . Hence the conflict will not happen.